tools/code-owners-parser/CodeOwnersParser/MatchedCodeownersEntry.cs

```diff
@@ -52,20 +52,11 @@ internal class MatchedCodeownersEntry
         /// </summary>
         private static readonly char[] unsupportedChars = { '[', ']', '!', '?' };
 
-        private readonly ILogger<MatchedCodeownersEntry> log;
```

Comment:
Is the ILogger interface from the other package you removed? If we ever want to redirect this logging to a file or some other service we might need to reintroduce a logging abstraction of some sort.

Reply:
@weshaggard yes, this is the logger I added recently, and now removed per [our discussion](https://github.com/Azure/azure-sdk-tools/pull/5063#discussion_r1066019443). Basically I am following YAGNI / You Aren't Gonna Need It principle here. If we need more advanced logging, then I'll re-add back the logger.